include/MVC/Controller/entry_point_registry.php

```diff
@@ -98,4 +98,5 @@
     'sendConfirmOptInEmail' => array('file' => 'include/entryPointConfirmOptInConnector.php', 'auth' => true),
     'saveGoogleApiKey' => array('file' => 'modules/Users/entryPointSaveGoogleApiKey.php', 'auth' => true),
     'setImapTestSettings' => ['file' => 'include/Imap/ImapTestSettingsEntry.php', 'auth' => true],
+    'queryWebServerFromCLI' => ['file' => 'include/entryPointQueryWebServerFromCLI.php', 'auth' => false],
```

Comment:
Hi @pgorod, what is the need to have this a non-authenticated.
I would prefer to not have it. The less information we expose the better.

Reply:
Hi @pgorod, I think I understand it better now. The idea is to get the info from the webserver, which may be different from the cli one.